### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by austinandrade**
# Description

I've added an in-depth readme including the following sections:
- About the Project
- Getting Started
- Endpoints
- Roadmap
- Contributing
- Contributors
I'd love insight or any proposed tweaks! Maybe we add a public postman list with all localhost:3000 endpoints to make requests to the dumby data?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,10 +202,72 @@ headers:
...
+1. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+2. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+3. Push to the Branch (`git push origin feature/AmazingFeature`)
+4. Open a Pull Request from your fork to this main repo
+
+<!-- CONTRIBUTORS -->
+### Contributors
+##### Backend Team
+Austin Andrade [![Github][github-shield]][austin-github-url] [![LinkedIn][linkedin-shield]][austin-linkedin-url] [![Email][email-shield]][austin-email-url] <br>
+Aidan Mcguire [![Github][github-shield]][aidan-github-url] [![LinkedIn][linkedin-shield]][aidan-linkedin-url] [![Email][email-shield]][aidan-email-url] <br>
```
Comment (ghost) [inline README.md]: Capitalize the 'G' in McGuire for me please 🤠

**Review by ghost — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,31 +1,77 @@
...
+  
+</details>
+<!-- ABOUT THE PROJECT -->
 
-* Ruby version
+## About The Project
+Head to Toe is a publically sourced LGBTQ+ health professional referral application. Find peer-acknowledged doctor and mental health professionals care across the U.S. Or, submit a resource for review and help others find care near them.
+<br> 
+<br>
+The Head to Toe backend is an API that aggregates doctor and mental health professional data to its partner frontend application. The frontend repo is located [here](https://github.com/Head-To-Toe/headToToe).
```
Comment (ghost) [inline README.md:36]: Maybe I am blind but we should include a link to the deployed FE

File: README.md [hunk 3]
```diff
@@ -1,31 +1,77 @@
...
+* [Factory Bot](https://github.com/thoughtbot/factory_bot)
+* [Faker](https://github.com/faker-ruby/faker)
+* [Fast JSON API](https://github.com/Netflix/fast_jsonapi)
+* [Rack Throttle](https://github.com/dryruby/rack-throttle)
 
-* Configuration
+* [RSpec](https://rspec.info/)
+* [Simplecov](https://github.com/simplecov-ruby/simplecov)
+* [Capybara](https://github.com/teamcapybara/capybara)
+* [Shoulda-Matchers](https://github.com/thoughtbot/shoulda-matchers)
```
Comment (ghost) [inline README.md]: Maybe put these in line over 2-3 lines, or centered so they don't take up so much space?

File: README.md [hunk 4]
```diff
@@ -1,31 +1,77 @@
...
+4. Create PostgreSQL database
+   ```sh
+   rake db:{create,migrate,seed}
+   ```
+5. Initialize rails server
+   ```sh
+   rails s
+   ```
+<!-- Endpoints -->
 ## Endpoints
```
Comment (ghost) [inline README.md:61]: We'll want to include info about rate limiting

File: README.md (hunk 2 above)
Comment (austinandrade) [reply README.md:36]: > Maybe I am blind but we should include a link to the deployed FE

Line 36 here covers that :D

File: README.md (hunk 3 above)
Comment (austinandrade) [reply README.md]: > Maybe put these in line over 2-3 lines, or centered so they don't take up so much space?

That makes sense. I will make that adjustment :)

File: README.md (hunk 1 above)
Comment (austinandrade) [reply README.md]: > Capitalize the 'G' in McGuire for me please 🤠

Noted!

**Pull request merged**